tests/features/decompose.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Decompose trait test
+
+Scenario Outline: Compare newly split yuv channels to expected values
+  Given the image '<full_image>' is loaded
+  When comparing yuv channels
+  Then check split success
```

Comment:
```suggestion
  Then the split channels match the references
```

then steps should be formulated in passive, but this is just  a hint - I can fix this later :)